tests/ui/macros/missing-writer.rs

```diff
@@ -0,0 +1,11 @@
+//Check error for missing writer in writeln! and write! macro
+fn main() {
+    let x = 1;
+    let y = 2;
+    write!("{}_{}", x, y);
+    //~^ ERROR format argument must be a string literal
+    //~| ERROR cannot write into `&'static str`
+    writeln!("{}_{}", x, y);
+    //~^ ERROR format argument must be a string literal
+    //~| ERROR cannot write into `&'static str`
+}
```

Comment:
Similarly not important, but files typically end in a newline
```suggestion
}

```